src/runtime/EncodedValue.h

```diff
@@ -59,13 +64,37 @@ class DoubleInEncodedValue : public gc {
 
     double value()
     {
+#ifdef ESCARGOT_32
+        double ret;
+        uint32_t* buf = reinterpret_cast<uint32_t*>(&ret);
+        buf[0] = m_buffer[0];
+        buf[1] = m_buffer[2];
```

Comment:
For 32bit platform, double value is separately stored in buffer.
Would these two store/load operations slow down the overall EncodedValue perforamance?

Reply:
you are right. dividing saving space causes slow down of copy(little)
but, If save tag in middle of DoubleInEncodedValue,
we can find it is a {double, object, pointer} with one reading from memory.